### ISSUE TITLE
The Demo doesn't work: Xcode 9.3, Swift 4.1

### ISSUE THREAD

**Issue by richardtop** [onmyway133/UsingPlayground#1]
Hi Khoa, thanks for the article on Medium.

Unfortunately, the project doesn't run out-of-the box and crashes with the following error:
```Swift
Playground execution failed:

error: Couldn’t lookup symbols:
__T012AppFramework22ResourceViewControllerCMa

```
Hope, you fix it soon, as overall, it's a very interesting concept.

**Comment by onmyway133** [comment]
@richardtop Hei, what version of CocoaPods are you using?

**Comment by richardtop** [comment]
Hi, I'm using CocoaPods `1.5.0`

**Comment by onmyway133** [comment]
@richardtop I mentioned that in my article

**Comment by richardtop** [comment]
Oh, didn't see that part, cause I got stuck with that error right at the beginning. I suggest moving that note about CocoaPods 1.5 at the beginning of the article, since it's the current stable version now.

**Comment by onmyway133** [comment]
@richardtop Hi, just saw someone made an issue https://github.com/CocoaPods/CocoaPods/issues/7689. Hope this get resolved soon

**Comment by toddgower-hiatus** [comment]
It is fixed in Cocoapods `1.5.3`. Perhaps the gemfile can be removed.
